libcxx/include/ranges

```diff
@@ -460,7 +454,7 @@ namespace std {
 // [ranges.syn]
 #  include <compare>
 #  include <initializer_list>
-#  include <iterator>
+#  include <iterator> // some CPOs are provided in <iterator> per [range.access.general]
```

Comment:
I don't think we should rely on that. Is there any good reason to remove the includes above?

Reply:
Reverted. I originally thought that it could be better to avoid seemingly awkward duplicated inclusion.